### PR TITLE
Add Wolfram Alpha extension

### PR DESCRIPTION
**Pull request by mathieudutour**
## Description

 Small extension to query [Wolfram Alpha](https://www.wolframalpha.com) and see its result in Raycast

## Type of change

- New extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/wolfram-alpha/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      "name": "index",
+      "title": "Query Wolfram Alpha",
+      "description": "Template for showing a static list and a detail view",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "appId",
+      "type": "textfield",
```
Comment (pernielsentikaer) [inline extensions/wolfram-alpha/package.json:20]:
```suggestion
      "type": "password",
```
Could be a password?

File: extensions/wolfram-alpha/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        i: query,
+        width: `${474 * 2}`,
+        units: preferences.units || "metric",
+        fontsize: `${14 * 2}`,
+      });
+      if (theme) {
+        params.append("background", "transparent");
+        params.append("foreground", theme === "light" ? "black" : "white");
+      }
+      const res = await fetch(`http://api.wolframalpha.com/v1/simple?${params.toString()}`);
```
Comment (pernielsentikaer) [inline extensions/wolfram-alpha/src/index.tsx:49]:
```suggestion
      const res = await fetch(`https://api.wolframalpha.com/v1/simple?${params.toString()}`);
```
Without `https` it fails

File: extensions/wolfram-alpha/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <SubmitFormAction onSubmit={onSubmit} title="Query" icon={Icon.MagnifyingGlass} />
```
Comment (pernielsentikaer) [inline extensions/wolfram-alpha/src/index.tsx:82]: Maybe a check here so its only visible in the form?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @mathieudutour 👋 

Thanks for an awesome extension, Wolfram is awesome!

I left a few suggestions – there is also something funny happening with the Query action, see this video

https://user-images.githubusercontent.com/38033107/150628510-fe82079c-1e90-44ae-b2c0-a16180c70ee8.mp4

Looking forward to testing it again! Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by mathieudutour** [comment]
> there is also something funny happening with the Query action

Yeah I saw that, that's a bug IMO so I'll see how to fix it

File: extensions/wolfram-alpha/src/index.tsx (hunk 2 above)
Comment (mathieudutour) [reply extensions/wolfram-alpha/src/index.tsx]: Oh they changed it on Friday! It used to work on Friday morning, not anymore haha

File: extensions/wolfram-alpha/package.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+          "title": "Imperial",
+          "value": "imperial"
+        }
+      ],
+      "description": "The system of units to use for measurements and quantities"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.26.0",
+    "macos-audio-devices": "^1.4.0",
```
Comment (pernielsentikaer) [inline extensions/wolfram-alpha/package.json:46]:
```suggestion
```
Didn't see this the first time, should be removed

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @mathieudutour 💪 

Besides the problem with the stuck “Query” button (Not happening all the time) and the one new suggested, I think we're ready to move!

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/wolfram-alpha/package.json (hunk 4 above)
Comment (mathieudutour) [reply extensions/wolfram-alpha/package.json]: ah yep, that's the issue when I work on too many things at the same time 😄

**Review by pernielsentikaer — APPROVED** [review]
Awesome @mathieudutour 💪  Much better!

**Pull request merged**